index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- Main Css Link -->
+    <link rel="stylesheet" href="styles/main.css">
+
+    <title>My Portfolio</title>
+</head>
+
+<body>
+
+    <header>
+        <a href="#" class="logo"> Welcome </a>
+
+        <nav>
+            <div>
+                <img src="./images/menu.svg" alt="menu-icon" class="menu">
+            </div>
+        </nav>
+
+    </header>
+
+    <section class="portfolio">
+        <h1 class="portfolio__title">I’m Ahmed Eid. Glad to see you!</h1>
+
+        <p class="portfolio__description">
+            I'm graduated from the Faculty of Computer and Informatics at Suez Canal University, where I majored
+            in Computer Science. My primary area of interest is web development, with a particular focus on the Backend
+            track.
+        </p>
+        <h2 class="contact__title">LET'S CONNECT</h2>
+        <ul class="portfolio__icons">
+            <li>
+                <a href="https://twitter.com/ahmedeid2684" rel="noopener noreferrer" target="_blank">
+                    <img src="images/twitter.svg" alt="twitter-logo">
+                </a>
+            </li>
+            <li>
+                <a href="https://www.linkedin.com/in/ahmed-eid-0018571b1/" rel="noopener noreferrer" target="_blank">
+                    <img src="images/linkedin.svg" alt="linkedin-logo">
+                </a>
+            </li>
+            <li>
+                <a href="https://github.com/ahmedeid6842" rel="noopener noreferrer" target="_blank">
+                    <img src="images/github.svg" alt="github-logo">
+                </a>
+            </li>
+        </ul>
```

Comment:
- [ ] [OPTIONAL] I have noticed that you are missing the `medium` and `Angelist` connections. Kindly consider adding these social links so that the recruiters can easily access all your social medias to learn more about your skills. This will increase your chances of getting hired ☺️.